src/dex-ui/services/HederaService/GovernorService.ts

```diff
@@ -218,10 +222,20 @@ interface ExecuteProposalParams {
  * @returns
  */
 const executeProposal = async (params: ExecuteProposalParams) => {
-  const { contractId, title, signer } = params;
+  const { contractId, title, signer, transfersFromAccount, tokenId, tokenAmount } = params;
   const governorContractId = ContractId.fromString(contractId);
   /** This parameter is named 'description' on the contract function */
   const contractFunctionParameters = new ContractFunctionParameters().addString(title);
+
+  if (tokenId && transfersFromAccount && tokenAmount) {
```

Comment:
This will be false if `tokenAmount` is 0 - is that intended?

Reply:
Yeah true, I am not sure if creating a proposal with 0 token amount is valid case or not but it will fail anyway for the user while executing the proposal let me fix this.

Reply:
I have changed the condition from `min: { value: 0, message: "A token amount must be greater than 0." },` to `validate: (value) => value > 0 || "A token amount must be greater than 0.",` I guess thats the correct condition in `TokenTransferProposalForm`.  let me know if this works or else we can put `!== undefined` condition for token amount.